.github/workflows/test-build.yml

```diff
@@ -0,0 +1,18 @@
+name: Build site
```

Comment:
Is this idea behind having this in a separate file to run this as part of a PR? If yes I like that :)

Reply:
Yes! To be fair, we already had the same PR build, I've just refactored a bit. :-)